src/lib/main.dart

```diff
@@ -1,33 +1,43 @@
+import 'package:core/core.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:infrastructure/infrastructure.dart';
+import 'package:jebud_app/pages/home_page.dart';
+import 'package:jebud_app/pages/splash_screen.dart';
+import 'pages/initiation_page.dart';
 
-void main() {
-  runApp(const MyApp());
-}
-
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+void main() async {
+  var budgetRepository = BudgetRepositoryImpl("jebud_db");
+  await budgetRepository.init();
 
-  @override
-    Widget build(BuildContext context) {
-      return MaterialApp(
-          theme: ThemeData(
-            primarySwatch: Colors.blue,
-            ),
-          home: const MyHomePage(),
-          );
-    }
+  runApp(
+      MyApp(() => BudgetManagerBloc(budgetRepository, DateTimeServiceImpl())));
```

Comment:
Styling is not correct.
```suggestion
  runApp(MyApp(() => BudgetManagerBloc(budgetRepository, DateTimeServiceImpl())));
```